lib/jekyll/cloudinary.rb

```diff
@@ -11,161 +10,196 @@ def initialize(tag_name, markup, tokens)
-        settings = site.config['cloudinary']
+        url = site.config["url"]
+        settings = site.config["cloudinary"]
 
         # Get Markdown converter
         markdown_converter = site.find_converter_instance(::Jekyll::Converters::Markdown)
 
         preset_user_defaults = {}
-        if settings['presets']
-          if settings['presets']['default']
-            preset_user_defaults = settings['presets']['default']
+        if settings["presets"]
+          if settings["presets"]["default"]
+            preset_user_defaults = settings["presets"]["default"]
           end
         end
 
         preset = preset_defaults.merge(preset_user_defaults)
 
         # Render any liquid variables in tag arguments and unescape template code
-        rendered_markup = Liquid::Template.parse(@markup).render(context).gsub(/\\\{\\\{|\\\{\\%/, '\{\{' => '{{', '\{\%' => '{%')
+        rendered_markup = Liquid::Template.parse(@markup)
+          .render(context).gsub(%r!/\\\{\\\{|\\\{\\%/!, '\{\{' => "{{", '\{\%' => "{%")
 
         # Extract tag segments
-        markup = /^(?:(?<preset>[^\s.:\/]+)\s+)?(?<image_src>[^\s]+\.[a-zA-Z0-9]{3,4})\s*(?<html_attr>[\s\S]+)?$/.match(rendered_markup)
-        if !markup
-          Jekyll.logger.abort_with('[Cloudinary]', "Can't read this tag: #{@markup}")
+        markup =
+          %r!/^(?:(?<preset>[^\s.:\/]+)\s+)?(?<image_src>[^\s]+\.[a-zA-Z0-9]{3,4})\s*(?<html_attr>[\s\S]+)?$/!
+            .match(rendered_markup)
+        unless markup
+          Jekyll.logger.abort_with("[Cloudinary]", "Can't read this tag: #{@markup}")
         end
 
         image_src = markup[:image_src]
 
         # Build source image URL
-        is_image_path_absolute = /^\/.*$/.match(image_src)
+        is_image_path_absolute = %r!/^\/.*$/!.match(image_src)
         if is_image_path_absolute
-          image_path = File.join(site.config['destination'], image_src)
+          image_path = File.join(site.config["destination"], image_src)
           image_url = File.join(url, image_src)
         else
-          image_path = File.join(site.config['destination'], File.dirname(context['page'].url), image_src)
-          image_url = File.join(url, File.dirname(context['page'].url), image_src)
+          image_path = File.join(
+            site.config["destination"],
+            File.dirname(context["page"].url),
+            image_src
+          )
+          image_url = File.join(url, File.dirname(context["page"].url), image_src)
         end
 
         # Get source image natural width
         if File.exist?(image_path)
-          image = Magick::Image::read(image_path).first
+          image = Magick::Image.read(image_path).first
           natural_width = image.columns
-          fallback_url = "https://res.cloudinary.com/#{settings['cloud_name']}/image/fetch/c_scale,w_#{natural_width},q_auto,f_auto/#{image_url}"
+          fallback_url = "https://res.cloudinary.com/\
+          #{settings["cloud_name"]}/image/fetch/c_scale,w_\
```

Comment:
I find lisibility suffers from this, maybe it would be better with indentations?